cg/server/api.py

```diff
@@ -276,15 +279,17 @@ def pools():
     customer_objs: Optional[Customer] = (
         None if g.current_user.is_admin else g.current_user.customers
     )
-    pools_q = db.pools(customers=customer_objs, enquiry=request.args.get("enquiry"))
-    data = [pool_obj.to_dict() for pool_obj in pools_q.limit(30)]
-    return jsonify(pools=data, total=pools_q.count())
+    pools_q: List[Pool] = db.get_pools_for_customer(
+        customers=customer_objs, enquiry=request.args.get("enquiry")
+    )
+    data = [pool_obj.to_dict() for pool_obj in pools_q[:30]]
```

Comment:
```suggestion
    data: List[Dict] = [pool_obj.to_dict() for pool_obj in pools_q[:30]]
```
Also the 30 literal here